tests/validate_yaml_load.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
+
+# Basic yaml syntax check
+yaml_files=$(find -iregex '.*\.\(yaml\|yml\)$')
+for yaml_file in $yaml_files; do
+    echo "$yaml_file"
+    /usr/bin/env python3 -c 'import yaml,sys;yaml.safe_load(sys.stdin)' < $yaml_file
```

Comment:
A solid question.  There will always be yamls in the repo, as the tooling is built around yamls and this repo is where those files live.  :tropical_fish: